### ISSUE TITLE
[Bug]: Fail to delete cloud save from the slot #6. 

### ISSUE THREAD

**Issue by Dai-JiaLin** [MelvorIdle/melvoridle.github.io#2616]
### Describe the bug

As the title says, nothing happens after confirm at the popup warning, and the #6 cloud save still exists. However, local save can be deleted properly. 

### Reproduction Steps

Just create a new save in #6 and sync, then you can't delete it from cloud. 

### Screenshots

![image](https://user-images.githubusercontent.com/50546779/194827565-46151193-f5aa-4d88-b132-ca8bd0bf48b4.png)

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [x] Edge
- [ ] Safari
- [ ] Mobile App (iOS)
- [ ] Mobile App (Android)
- [x] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.0.5

### Game Subversion

?2129

### Are you using any scripts or extensions?

Yes

**Comment by Dai-JiaLin** [comment]
The bug exists after disabling all scripits and extensions.

**Comment by supersaiyoon** [comment]
I am also experiencing the same issue. I have a cloud save in slot #6 that still remains even after I go through the deletion process. I noticed this issue back when Hardcore Speedrun mode ended. I was able to delete the local save, but the cloud save won't go away even after I deleted it.

Thinking that it was due to the special limited time nature of that game mode, I created a new normal character then saved that game in the cloud to overwrite the original cloud save. The cloud file still won't disappear after deleting.
